wirecard/oxid-ee
# WDPD-24: final paypal notification

### This PR
* Adapts db tables
* Adds transaction data to ordertransaction table in db
* Adds additional data to order table in db
* Adds order details tab to admin order view

### TODO
* Add unit tests
* Adapt Paypal response handling as soon as WDPD-23 is merged (update oxid order number in transaction table)

### How to Test
* Buy something with wirecard paypal payment method
* Login to the admin to check the state in the transaction table(Wirecard Transactions->Transactions)
* From after finishing the order till before finishing paypal payment the transaction state should be 'awaiting'
* After finishing the paypal payment the state should be changed to 'success' for transaction 

### Jira Links
* https://jira.parkside.at/browse/WDPD-24
* https://jira.parkside.at/browse/WDPD-101
